app/web/service/admin.ts

```diff
@@ -0,0 +1,49 @@
+import { Timestamp } from "google-protobuf/google/protobuf/timestamp_pb";
+import {
+  GetUserDetailsReq,
+  GetUserReq,
+  ListUserIdsReq,
+  UserDetails,
+} from "proto/admin_pb";
+import { User } from "proto/api_pb";
+
+import client from "./client";
+
+export async function getUser(user: string): Promise<User.AsObject> {
+  const req = new GetUserReq();
+  req.setUser(user || "");
+  return (await client.admin.getUser(req)).toObject();
+}
```

Comment:
Don't we already have a `useUser` hook?

Reply:
This uses `admin.getUser` which bypasses banned etc